server/service/orbit_client.go

```diff
@@ -146,6 +146,20 @@ func (oc *OrbitClient) SetOrUpdateDeviceToken(deviceAuthToken string) error {
 	return nil
 }
 
+// SetOrUpdateDeviceMappingEmail sends a request to the server to set or update the
+// device mapping email with the given value.
+func (oc *OrbitClient) SetOrUpdateDeviceMappingEmail(email string) error {
+	verb, path := "POST", "/api/fleet/orbit/device_mapping"
+	params := orbitPostDeviceMappingRequest{
+		Email: email,
+	}
+	var resp setOrUpdateDeviceTokenResponse
```

Comment:
Not that it's super critical since we don't care about the response besides an error, but this should be `orbitPostDeviceMappingResponse`.

Reply:
Thanks, I noticed this on our call!